src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,128 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int RESIZE_FACTOR = 2;
+    private static final int EMPTY_SIZE = 0;
+    private static final int NULL_KEY_HASHCODE = 0;
+    private int size;
+    private Node<K, V>[] table;
+    private int threshold;
+
+    private static class Node<K, V> {
+        private final K key;
+        private int hash;
+        private V value;
+        private Node<K, V> next;
+
+        public Node(K key, V value) {
```

Comment:
```suggestion
        Node(K key, V value) {
```
```suggestion
        private Node(K key, V value) {
```
won't be public anyway